### PR TITLE
adding polygon card

### PR DESCRIPTION
**Pull request by BrianS111**
Closes https://github.com/relaycc/relay/issues/41

### CODE REVIEW

**Comment by BrianS111** [comment]
https://user-images.githubusercontent.com/81312527/197067701-1c71c30f-09cb-4be8-899f-8e760a3b1685.mov

This PR is ready to be merged

File: src/components/pages/Profile.tsx [hunk 1]
```diff
@@ -615,6 +594,43 @@ export const Profile = ({ handle }: { handle?: string | null }) => {
...
+        }
+      />
+      <HoverToggle
+        fadeOut={<LogoCard title="PolygonId" logo={"/polygonid.svg"} />}
+        fadeIn={
+          <AppCard
+            onClickLogo={openInNewTab("https://polygon.technology/polygon-id/")}
+            title="PolygonId"
+            logo="/polygonid.svg"
+            logoAlt="Polydon Id Logo"
```
Comment (killthebuddh4) [inline src/components/pages/Profile.tsx:625]: Typo.

File: src/components/pages/Profile.tsx [hunk 2]
```diff
@@ -615,6 +594,43 @@ export const Profile = ({ handle }: { handle?: string | null }) => {
...
+            logoAlt="Disco Logo"
+            onClickLinkOut={openInNewTab("https://disco.xyz/")}
+            onClickSendMessage={onClickSendMessage()}
+            linkOutText={"View on Disco"}
+            logoClassName="invert"
+          />
+        }
+      />
+      <HoverToggle
+        fadeOut={<LogoCard title="PolygonId" logo={"/polygonid.svg"} />}
```
Comment (killthebuddh4) [inline src/components/pages/Profile.tsx:619]: The display string for this should have a space between "polygon" and "id". See 
image

**Review by killthebuddh4 — CHANGES_REQUESTED** [review]

**Comment by BrianS111** [comment]
https://user-images.githubusercontent.com/81312527/197252231-9a7a9501-8d47-4a3f-9bf1-20a561735001.mov

This issue should now be complete

**Comment by BrianS111** [comment]
Typo is fixed. This should be ready to be merged

**Review by killthebuddh4 — APPROVED** [review]

**Pull request merged**